File: message-api-schema-validator/src/main/java/uk/ac/jisc/rdss/tests/metadata/update/MetadataUpdateRequestTest.java
package uk.ac.jisc.rdss.tests.metadata.update;

import org.junit.Test;

import uk.ac.jisc.rdss.tests.AbstractSchemaValidatorTest;

public class MetadataUpdateRequestTest extends AbstractSchemaValidatorTest {

    @Override
    protected String getSchemaFileName() {
        return "messages/metadata/update/request_schema.json";
    }

    @Test
    public void validateRequest() throws Exception {
        validateJson("messages/metadata/update/request.json");
    }
}

File: message-api-schema-validator/tests/metadata/metadata_update_request_test.py
from tests.abstract_schema_validator_test import AbstractSchemaValidatorTest
from unittest import TestCase


class MetadataUpdateRequestTest(AbstractSchemaValidatorTest, TestCase):
    def get_json_schema_file_name(self):
        return 'messages/metadata/update/request_schema.json'

    def runTest(self):
        self.validate_json('messages/metadata/update/request.json')
